GUI/window.py

```diff
@@ -258,32 +373,35 @@ def run_micro_instructions(self, instance):
                     if self.first_inst:
                         self.inst_table.data_list.clear()
                         self.header = False
-                        self.inst_table.get_data(self.micro_sim.index, self.header, self.micro_sim.disassembled_instruction())
+                        self.inst_table.get_data(self.micro_sim.index, self.header,
+                                                 self.micro_sim.disassembled_instruction())
                         self.header = True
-                        self.inst_table.get_data(self.micro_sim.index, self.header, self.micro_sim.disassembled_instruction())
+                        self.inst_table.get_data(self.micro_sim.index, self.header,
+                                                 self.micro_sim.disassembled_instruction())
```

Comment:
Entiendo que esto es modalidad de Pep-8, pero quiero saber cuando es que es prudente hacerlo y cuando no (para aplicarlo)

Reply:
I use Pycharms/Visual Studio code autoformatter

Reply:
Got it.